### DISCUSSION TITLE
V2打开的时候经常会退出

### DISCUSSION THREAD

**Discussion by Jife** [cangzhang/ChampR#260] [General]
V2打开的时候经常会退出，是怎么回事呢？

**Comment by cangzhang** [comment]
你的系统是什么版本？

**Reply by Jife** [reply]
Win10 22H2

**Reply by Jife** [reply]
故障存储段 1774123225340085113，类型 5
事件名称: BEX64
响应: 不可用
Cab ID: 0

问题签名:
P1: ChampR.exe
P2: 2.0.0.0
P3: 64328bc6
P4: ChampR.exe
P5: 2.0.0.0
P6: 64328bc6
P7: 0000000000228c36
P8: c0000409
P9: 0000000000000007
P10:

**Comment by starolala** [comment]
我也是，能打开，但是打开后就是有一个绿色的箭头一直在转圈，然后就会自己退出
